lib/commander/help_formatters/terminal/help.erb

```diff
@@ -1,3 +1,6 @@
+<% unless @default_command.nil? -%>
```

Comment:
Hmm, I wonder if the logic to set `default_command` here should be moved to the base help formatter, especially as it's duplicated between the two help format files.